ButtonApp/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
It looks like you did some exploration around how styles/colors are defined in Android resources. When I first launched the app I had to do a bit of poking around to see why it looked dim. Keep in mind the importance of contrast when designing activities. The other thing about having a "dim" background on android is it looks kinda the same as what you would see if a "popover" is displayed over an activity (the portion of the activity still visible under the popover is dimmed to provide visual separation between the content and indicate that the user should focus on the content in the popover).